.drone.yml

```diff
@@ -85,3 +85,24 @@ pipeline:
     when:
       branch: master
       event: push
+
+  unit_tests:
+    image: 795250896452.dkr.ecr.us-east-1.amazonaws.com/server-tig/${DRONE_REPO_NAME}:latest
```

Comment:
I asked the kanopy team and they said after the dronev1 upgrade we should be able to use the local Dockerfile to build the image, so this would change to:
```
image: .
```

Reply:
(This actually may take longer to roll out than I initially said in this comment ^)